src/mesh/mesh.cpp

```diff
@@ -270,14 +270,12 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
   tree.CreateRootGrid();
 
   // Load balancing flag and parameters
-#ifdef MPI_PARALLEL
   if (pin->GetOrAddString("loadbalancing", "balancer", "default") == "automatic")
     lb_automatic_ = true;
   else if (pin->GetOrAddString("loadbalancing", "balancer", "default") == "manual")
     lb_manual_ = true;
   lb_tolerance_ = pin->GetOrAddReal("loadbalancing", "tolerance", 0.5);
   lb_interval_ = pin->GetOrAddReal("loadbalancing", "interval", 10);
```

Comment:
I'm kind of surprised by this logic (but it's also in Athena++).
Looks like the "default" means neither "automatic" nor "manual". So if "automatic" is not default, what is?
I'll create an issue as it is unrelated to this PR.